include/uv.h

```diff
@@ -1531,7 +1530,8 @@ struct uv_loop_s {
   /* Loop reference counting. */
   unsigned int active_handles;
   void* handle_queue[2];
-  void* active_reqs[2];
+  unsigned int active_req_count;
```

Comment:
This should probably be a `union { void* unused[2]; unsigned int active_req_count; }` to ensure it doesn't change the struct size or field offsets.

(As well, I'd call it just `active_reqs` for parity with `active_handles`.)

Reply:
all comments fixed.

I went with `active_reqs.count`, to avoid adding an unnamed union. On libuv v2, it can be  just `active_reqs` (and maybe also reorder the struct to put it with `active_handles`).

```
sed -i'' -s 's/active_reqs\.count/active_reqs/g' `find . -name '*.[ch]'`
```